admin/app/components/solidus_admin/orders/cart/result/component.js

```diff
@@ -0,0 +1,8 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  submit(event) {
+    debugger
```

Comment:
debugging leftover

Reply:
Thanks!  🤦‍♂️ 
We should have a linter for these things 🤔

Reply:
See https://github.com/solidusio/solidus/pull/5457